cocos/renderer/core/gfx/GFXDevice.h

```diff
@@ -104,6 +104,8 @@ class CC_CORE_API GFXDevice : public Object {
     bool _reverseCW = false;
     uint _shaderIdGen = 0;
     std::unordered_map<String, String> _macros;
+    GFXFormat _colorFmt = GFXFormat::UNKNOWN;
+    GFXFormat _depthStencilFmt = GFXFormat::UNKNOWN;
```

Comment:
Where are they initialized? `_context` may change, for example, after returning from background on Android, GL ES  context may be recreated.

Reply:
they are initialized in XXXDevice.xx source file